package.json

```diff
@@ -23,7 +23,7 @@
   "author": "GitHub",
```

Comment:
Does the `version` field also need to be bumped up?

Reply:
Yes! Done!

Reply:
The change is not reflecting here. You may have to update the PR branch.